cgmes/cgmes-conversion/src/test/java/com/powsybl/cgmes/conversion/CgmesConversionContextExtensionTest.java

```diff
@@ -25,6 +25,7 @@ class CgmesConversionContextExtensionTest {
     void test() {
         Properties properties = new Properties();
         properties.put(CgmesImport.STORE_CGMES_CONVERSION_CONTEXT_AS_NETWORK_EXTENSION, "true");
+        properties.put(CgmesImport.IMPORT_ASSEMBLED_AS_SUBNETWORKS, "false");
```

Comment:
Have you run all the test with `true` behaviour? I understand that for performance issue, we prefer to run unit tests with false, but when I run them in local with true, I fall into errors.

Reply:
some expected results in the unit tests were only valid if we import with the "single network" configuration. I do not think we have to adapt old unit tests to the new feature. We have already added specific unit tests for the new feature